Library/Formula/symlinks.rb

```diff
@@ -0,0 +1,23 @@
+class Symlinks < Formula
+  desc "scan/change symbolic links"
+  homepage "http://www.ibiblio.org/pub/Linux/utils/file/symlinks.lsm"
```

Comment:
Neither, _specifically_. It's just something that'd be nice to check, since it's a tiny easy thing to look into and we're not yet stopping people from doing:
```
brew install turtles --env=std
```
To completely bypass superenv, and then throwing Issues around about it, which we either have to close and annoy said user or fix and annoy ourselves for not checking in the first place.